custom_components/kia_uvo/binary_sensor.py

```diff
@@ -300,6 +300,162 @@ class HyundaiKiaBinarySensorEntityDescription(BinarySensorEntityDescription):
         device_class=BinarySensorDeviceClass.WINDOW,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    HyundaiKiaBinarySensorEntityDescription(
+        key="accessory_on",
+        name="Accessory",
+        is_on=lambda vehicle: getattr(vehicle, "accessory_on", None),
```

Comment:
getattr shouldn't be needed.  The key is checked for a value and the entity is only added if it exists.  Same applies for the others.